src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -97,9 +97,9 @@ public final class Constants {
     public static final long MAX_SUBMISSION_FILE_SIZE = 8 * 1024 * 1024; // 8 MB
 
     // Note: The values in input.constants.ts (client) need to be the same
-    public static final int MAX_SUBMISSION_TEXT_LENGTH = 30 * 1000; // 30.000 characters
+    public static final int MAX_SUBMISSION_TEXT_LENGTH = 30_000; // 30.000 characters
 
-    public static final int MAX_SUBMISSION_MODEL_LENGTH = 100 * 1000; // 100.000 characters
+    public static final int MAX_SUBMISSION_MODEL_LENGTH = 100_000; // 100.000 characters
```

Comment:
do we need that comment?

Reply:
It specifies that it is about chars. I don't see any harm from this comment either